package.json

```diff
@@ -37,22 +37,21 @@
   "homepage": "https://github.com/ipfs/js-datastore-s3#readme",
   "dependencies": {
     "datastore-core": "^1.1.0",
-    "interface-datastore": "^1.0.2",
-    "upath": "^1.1.0"
```

Comment:
@achingbrain did you mean to remove upath? It's still being used. CI is failing due to this

Reply:
Oops, I had some un-pushed changes.  We use `upath` to join s3 paths with platform-specific slashes/remove duplicate slashes which is probably overkill as s3 paths are all forward slashes, so I think we can just cut out the middleman and join/dedupe them ourselves.

`upath` also makes building a browser bundle fail with aegir 24.x.x as support for polyfilling node globals has been removed.